test/EndToEndTests/Tests/Client/Microsoft.OData.Client.E2E.Tests/Common/Client/Default/DefaultServiceCsdl.xml

```diff
@@ -146,9 +146,9 @@
         <Property Name="CountryRegion" Type="Edm.String" />
         <Property Name="AccountInfo" Type="Microsoft.OData.Client.E2E.Tests.Common.Server.Default.AccountInfo" />
         <Property Name="UpdatedTime" Type="Edm.DateTimeOffset" Nullable="false" />
-        <NavigationProperty Name="MyPaymentInstruments" Type="Collection(Microsoft.OData.Client.E2E.Tests.Common.Server.Default.PaymentInstrument)" />
-        <NavigationProperty Name="ActiveSubscriptions" Type="Collection(Microsoft.OData.Client.E2E.Tests.Common.Server.Default.Subscription)" />
-        <NavigationProperty Name="MyGiftCard" Type="Microsoft.OData.Client.E2E.Tests.Common.Server.Default.GiftCard" />
+        <NavigationProperty Name="MyPaymentInstruments" Type="Collection(Microsoft.OData.Client.E2E.Tests.Common.Server.Default.PaymentInstrument)" ContainsTarget="true" />

```

Comment:
Is there a reason for the navigation properties changing from non-contained to contained?

Reply:
I had missed this. This is also a shared csdl and will be required when testing `containment` tests